src/utilities/Kernel.js

```diff
@@ -65,3 +69,35 @@ export function animateKernel(id) {
     step(element, intervalId);
   }, KERNEL_FOUNTAIN_INTERVAL);
 }
+
+/**
+ * Initialize and return a reference to a randomized popcorn kernel element.
+ *
+ * @param {Document} documentElement
+ * @param {string} id
+ * @param {number} left
+ * @param {number} top
+ * @param {boolean} isRotatingClockwise
+ */
+export function getKernelElement(
+  documentElement,
+  id,
+  left,
+  top,
+  isRotatingClockwise,
+) {
+  const kernelElement = documentElement.createElement('img');
+
+  kernelElement.alt = 'Popcorn kernel';
+  kernelElement.className = isRotatingClockwise
+    ? 'popcorn-kernel'
+    : 'popcorn-kernel popcorn-kernel--reverse';
+  kernelElement.id = id;
+  kernelElement.src = kernel;
+  kernelElement.style = `display:'none';left:${left}px;top:${top}px;`;
+  kernelElement.setAttribute('aria-hidden', 'true');
+  kernelElement.setAttribute('data-x-velocity', getRandomInteger(-15, 15));
+  kernelElement.setAttribute('data-y-velocity', getRandomInteger(-20, -10));
+
+  return kernelElement;
```

Comment:
A bunch of this logic is duplicated across the existing React component / JSX implementation. Will likely follow up to de-duplicate in favor of this native technique that is probably more performant.